content/millennium/r4/proprietary-codes-and-systems.md

```diff
@@ -130,6 +130,12 @@ The ChargeItem Resource supports proprietary codes for:
 
 * The Bill Code Schedule extensions on ChargeItem are maintained in [Code Set 14002 Bill Code Schedule](#code-set-14002-bill-code-schedule)
 
+<h4>InsurancePlan</h4>
+
+The InsurancePlan Resource supports proprietary codes for:
+
+* InsurancePlan.type codes are maintained in [Code Set 27137 Insurance plan Type](#code-set-27137-insurance-plan-type)
```

Comment:
Code Set 27137 is named Health Plan Service Type

Reply:
Done https://github.com/cerner/fhir.cerner.com/pull/809/commits/5c88f0cc2c198cbd22158c547868aa2268b7827a#diff-aedba7f37b773d7a314400db492774b8a9d406ed43aad2101793b82970106103R717